libsolidity/ast/Types.cpp

```diff
@@ -2852,7 +2870,7 @@ bool FunctionType::canTakeArguments(TypePointers const& _argumentTypes, TypePoin
 		);
 }
 
-bool FunctionType::hasEqualArgumentTypes(FunctionType const& _other) const
+bool FunctionType::hasEqualParameterTypes(FunctionType const& _other) const
```

Comment:
Can you pull this change into a separate PR? Look like a good change we could merge easily.